kitchen.yml

```diff
@@ -0,0 +1,19 @@
+---
+driver:
+  name: vagrant
+  synced_folders:
+    - [".", "/opt/ipmininet"]
+
+provisioner:
+  name: shell
+  command: sudo python /opt/ipmininet/util/install.py -a
```

Comment:
```suggestion
  command: if which apt; then sudo apt install -y python; elif which yum; then sudo yum install -y python; fi; sudo python /opt/ipmininet/util/install.py -iam && sudo pytest -v /opt/ipmininet
```

Reply:
This solves the problems for all platforms

Reply:
:+1: for `if which apt; then sudo apt install -y python; elif which yum; then sudo yum install -y python; fi; sudo python /opt/ipmininet/util/install.py -iam`

For `sudo pytest -v /opt/ipmininet` see https://github.com/cnp3/ipmininet/pull/31#discussion_r288594278